bigbluebutton-html5/public/locales/en.json

```diff
@@ -230,7 +230,13 @@
     "app.presentation.presentationToolbar.goToSlide": "Slide {0}",
     "app.presentation.placeholder": "There is no currently active presentation",
     "app.presentationUploder.title": "Presentation",
-    "app.presentationUploder.message": "As a presenter you have the ability to upload any office document or PDF file.  We recommend PDF file for best results. Please ensure that a presentation is selected using the circle checkbox on the right hand side.",
+    "app.presentationUploder.message": "As a presenter you have the ability to upload any office document or PDF file.  We recommend PDF file for best results. Please ensure that a presentation is selected using the circle checkbox on the left hand side.",
+    "app.presentationUploader.exportHint": "Selecting \"Send to chat\" will provide users with a downloadable link with annotations in public chat.",
+    "app.presentationUploader.exportToastHeader": "Downloading to chat ({0} item)",
+    "app.presentationUploader.exportToastHeaderPlural": "Downloading to chat ({0} items)",
```

Comment:
Minor and subjective suggestion, but since we have "Send to chat" before we could have "Sending to chat" here. Personally I like "Share/sharing with chat" the most :)